llama_stack/distribution/server/server.py

```diff
@@ -278,6 +281,18 @@ def main(
         config = StackRunConfig(**yaml.safe_load(fp))
 
     app = FastAPI()
+    # instantiate kvstore for storing and retrieving distribution metadata
```

Comment:
ah no, I don't think we should do globals like this. we should always try to "pass dependencies from above" (lightweight dependency injection style)

- who needs this object? RoutingTable 
- who instantiates RoutingTable? resolve_impls
- we must pass this as an argument to resolve_impls() or have resolve_impls() instantiate the object based on config from run_config. we should have some way to configure this kvstore also, just  that the default is a sqlite with some default path like you have.

Reply:
done.